### DISCUSSION TITLE
Dense retrieval non-English languages support

### DISCUSSION THREAD

**Discussion by Fatima-Haouari** [castorini/pyserini#1163] [Q&A] [answered]
Hi,
I am interested in using the dense retrieval, and I wanted to know if only English models are supported as I am working on Arabic data.

**Comment by lintool** [comment, marked as answer]
Yes, we do have support for multilingual dense retrieval, but the features are not well exposed in Pyserini at present. See:
https://arxiv.org/abs/2204.02363

You might want to follow along here:
https://github.com/castorini/pyserini/pull/1117

**Reply by Fatima-Haouari** [reply]
Thanks for your reply. So it can not be used yet.
